jenkins/run

```diff
@@ -11,6 +11,7 @@
 # CONTAINER_HOME_OPTION [z] (used only if CONTAINER_HOME is given)
 # EXTRA_PODMAN_SWITCHES []
 # IMAGE_REPO [quay.io/pbench]
+# IMAGE_KIND [fedora] (see Makefile for options)
 #
 # For example, run with HOME mapped to /tmp/jenkins in the host system, and run
 # an interactive bash shell:
```

Comment:
Do we know why this script insists upon having a path which is the current working directory plus a `jenkins` subdirectory?  I _think_ it's because it wants to map the Git checkout on the host to `/src/pbench` inside the container.  If that's the case, it can do that based either on the return from `git rev-parse` or by analyzing its own path -- it shouldn't need to use `$(pwd)` for this.

Does anyone else think that we should relax the current working directory restriction, if we can?

Reply:
This is all going to be changed in PR #2551.  So not worth addressing here.